src/test/src/controllers/unincorporatedCorrespondenceAddressManualController.test.ts

```diff
@@ -0,0 +1,176 @@
+import mocks from "../../mocks/all_middleware_mock";
+import supertest from "supertest";
+import app from "../../../main/app";
+import { BASE_URL, UNINCORPORATED_CORRESPONDENCE_ADDRESS_MANUAL } from "../../../main/types/pageURL";
+
+jest.mock("@companieshouse/api-sdk-node");
+const router = supertest(app);
+
+describe("GET" + UNINCORPORATED_CORRESPONDENCE_ADDRESS_MANUAL, () => {
+    it("should return status 200", async () => {
+        await router.get(BASE_URL + UNINCORPORATED_CORRESPONDENCE_ADDRESS_MANUAL).expect(200);
+        expect(mocks.mockSessionMiddleware).toHaveBeenCalled();
+        expect(mocks.mockAuthenticationMiddleware).toHaveBeenCalled();
+    });
+});
+
+// Test for correct form details entered, will return 302 after redirecting to the next page.
+describe("POST" + UNINCORPORATED_CORRESPONDENCE_ADDRESS_MANUAL, () => {
+    xit("should return status 302 after redirect", async () => {
+        await router.post(BASE_URL + UNINCORPORATED_CORRESPONDENCE_ADDRESS_MANUAL)
+            .send({ addressPropertyDetails: "abc", addressLine1: "pqr", addressLine2: "pqr", addressTown: "lmn", addressCounty: "lmnop", addressCountry: "lmnop", addressPostcode: "MK9 3GB" }).expect(302);
+    });
+});
+// Test for no addressPropertyDetails, will return 400.
+describe("POST" + UNINCORPORATED_CORRESPONDENCE_ADDRESS_MANUAL, () => {
+    it("should return status 400", async () => {
+        await router.post(BASE_URL + UNINCORPORATED_CORRESPONDENCE_ADDRESS_MANUAL)
+            .send({ addressPropertyDetails: "", addressLine1: "pqr", addressLine2: "pqr", addressTown: "lmn", addressCounty: "lmnop", addressCountry: "lmnop", addressPostcode: "MK9 3GB" }).expect(400);
```

Comment:
add an expect to test the error message shows what is expected for each of the error scenarios